### ISSUE TITLE
Magnitude information for field maps?

### ISSUE THREAD

**Issue by Govish** [shimming-toolbox/susceptibility-to-fieldmap-fft#25]
Hi guys!

We were wondering if it's possible to get a hold of the original magnitude information used to make the field maps. We hope to use it to determine the shape/contour of the subject so we can place them appropriately in the coil. It's difficult to do this with just the field maps/masks as we have them now. 

Thanks!
-Ishaan

**Comment by jcohenadad** [comment]
@Govish sorry for the slow reply. Sure, we should be able to provide these. @CharlesPageot is this something you could help with?

**Comment by Govish** [comment]
Yeah, this is in the context of our current collaboration--let me know where a good place to continue the conversation. Thanks!

**Comment by jcohenadad** [comment]
> let me know where a good place to continue the conversation

I've just created [this repos](https://github.com/shimming-toolbox/shim-layout-optimization) and added you as admin-- please feel free to take it from there. Tagging @CharlesPageot @po09i @CharlesPageot @jaystock @evaalonsoortiz so everyone is aware

**Comment by CharlesPageot** [comment]
@Govish just to be sure, you are talking about the original magnitude images (the T1w) of the subjects? If that is the case, we should transfer to the [new repo](https://github.com/shimming-toolbox/b0-fieldmap-realistic-simulation) where there is already work being done to publish the data to OpenNeuro.

**Comment by Govish** [comment]
^ Yes--the original magnitude images (T1w) of the subjects would be great to have. Continuing in a different thread in the new repo works for me!
